sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCast.scala

```diff
@@ -419,6 +419,9 @@ case class GpuCast(
       case (from: DecimalType, to: DecimalType) =>
         castDecimalToDecimal(input.getBase, from, to)
 
+      case (_: DecimalType, StringType) =>
```

Comment:
I think we have to update the `CastChecks` to allow casting from decimal to string for this to actually impact user queries.

Reply:
Sorry. I forgot it.

Reply:
Fixed.